.github/workflows/ci-checks.yml

```
@@ -4,6 +4,10 @@ name: CI Check
 on:
   pull_request_target:
     types: [opened, synchronize, reopened, labeled]
+  workflow_run:
+    workflows: [ "Stable Merge Integration Test Check" ] # Triggers when "Stable Merge Integration Test Check" succeeds
+    types: [completed]
+    conclusions: [success]
```

actionlint (1.7.8)

unexpected key "conclusions" for "workflow_run" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)